cosmos/airflow/graph.py

```diff
@@ -166,6 +169,11 @@ def create_task_metadata(
             task_id = f"{node.name}_run"
             if use_task_group is True:
                 task_id = "run"
+            if model_timeout and "model_timeout" in node.config.keys():
+                logger.error(f'model_timeout: {node.config["model_timeout"]} in values')
+                args["execution_timeout"] = timedelta(seconds=int(node.config["model_timeout"]))
+            if model_sla and "model_sla" in node.config.keys():
+                args["sla"] = timedelta(seconds=int(node.config["model_sla"]))
```

Comment:
Airflow SLAs are known to have few issues. 

They mentioned in [AF 3 meeting notes](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+3+Dev+call%3A+Meeting+Notes) that they would be dropping SLAs of AF 3 & come up with something better in later versions in AF 3+. I am thinking if we should we hold on adding SLA support here? WDYT?
> It was agreed that for now, we will mark it for removal in Airflow 3.0 and adding it back in 3.1. We will assess again in the coming dev calls if something changes. Elad has also offered to help if needed.

Reply:
I understand well. I’ll consider SLA support to be out of scope. I wasn’t aware of the upcoming features planned for Airflow 3.0. Thank you for letting me know.